### ISSUE TITLE
Priority 2 accessibility issue: Focus element colour contrast

### ISSUE THREAD

**Issue by alicerichmond** [wellcomecollection/wellcomecollection.org#6741]
**What**

Non-text elements fall below the minimum colour contrast ratio requirement of 3:1, against adjacent colour.

The focus indicator colour used throughout the site falls below the minimum colour contrast ratio requirement of 3:1 due to the outline colour: #5cb8bf and background colour: FFFFFF having a contrast ratio of 2.3:1 and due to the use of transparency this is further lowered to a ratio of 1.8:1.

Solution:
Ensure that focus indicators meet the minimum colour contrast ratio of at least 3:1 against adjacent colour to provide a clearly visible focus indicator for users with vision impairments.
This issue occurs throughout the service.

Keyboard user:
 “I found that the pale blue focus highlighting was difficult to see throughout the majority of the website, a darker or better contrasting colour would improve the experience when navigating, making it easier to keep track of focus.”

**Where**

https://wellcomecollection.org

**Comment by gestchild** [comment]
@GarethOrmerod do you have a preference for the colour we should use instead?

**Comment by GarethOrmerod** [comment]
Looking at the focus state colour contrast against: white (#ffffff), charcoal (#323232), yellow (#ffce3c) and cream (#f0ede3) we current don't have a colour which has sufficient contrast.

The closest in our existing palette is cyan (#298187) which fails with 2.79:1 against charcoal. A very slight adjustment to #2B888F however does pass. I think this is also ok with the subtle shades used in the viewer.

We also have controls in the viewer which sit over images which is difficult to guarantee contrast for in every instance.
